string msgType
string agvId
int64 timestamp
int32 cmdId
int32 value
string ctrlType



